src/promptflow-evals/promptflow/evals/evaluators/coherence/__init__.py

```diff
@@ -12,12 +14,15 @@
 
 
 class CoherenceEvaluator:
-    def __init__(self, model_config: AzureOpenAIModelConfiguration):
+    def __init__(self, model_config: AzureOpenAIModelConfiguration,
+                 log_level: Optional[int] = None):

```

Comment:
a int parameter for customer is not a good idea. What's the possible value here?